VERSION

```diff
@@ -0,0 +1 @@
+v0.0.5
```

Comment:
Is this compatible with goreleaser which depends on git tag for versioning ?

Reply:
ha? I just took the latest release version here...

Reply:
Please refer to https://goreleaser.com/environment/ for getting version. We are using it to make releases in GitHub Actions.

> Default wise GoReleaser sets three ldflags:
> 
> * main.version: Current Git tag (the v prefix is stripped) or the name of the snapshot, if you're using the --snapshot flag
> * main.commit: Current git commit SHA
> * main.date: Date according RFC3339
> 
> You can use it in your main.go file:
> 
> ```
> package main
> 
> import "fmt"
> 
> var (
> 	version = "dev"
> 	commit  = "none"
> 	date    = "unknown"
> )
> 
> func main() {
>   fmt.Printf("%v, commit %v, built at %v", version, commit, date)
> }
> ```
> 
> You can override this by changing the ldflags option in the build section.
>

Reply:
checked goreleaser docs and got it, i'll do some changes in next PR